infra/base-images/base-runner/coverage

```diff
@@ -459,6 +460,44 @@ elif [[ $FUZZING_LANGUAGE == "jvm" ]]; then
     fi
   done
 
+  # Check if TARGET_PACKAGE_PREFIX has been set and remove classes not
+  # in the target package prefix to exclude them in the resulting HTML
+  # report. By default, Jacoco includes all classes in the classes dump
+  # directory to the resulting HTML report. Removing classes no in target
+  # package could concentrate the result on small subset of class and
+  # decrease the noise from project dependencies.
+  dir_to_remove=
+  have_target_package="NO"
+  if [[ -n $TARGET_PACKAGE_PREFIX ]]
+  then
+    for dir in $(find ./ -mindepth 1 -type d)
+    do
+      need_remove="YES"
+      for prefix in $(echo $TARGET_PACKAGE_PREFIX | sed "s#:# #g" | sed "s#\.#/#g" | sed "s#/\*##g")
+      do
+        if [[ "./$prefix" == "$dir"* ]]
+        then
+          have_target_package="YES"
+          need_remove="NO"
+        fi
+      done
+      if [[ "$need_remove" == "YES" ]]
+      then
+        dir_to_remove="$dir_to_remove$dir "
```

Comment:
Could you add a comment here just clarifying the ` ` at the end of the string

Reply:
added